arch/generic/crc32_c.c

```
@@ -0,0 +1,38 @@
+#include "zbuild.h"
+#include "crc32_braid_p.h"
+#include "crc32_c.h"
+
+uint32_t PREFIX(crc32_c)(uint32_t crc, const uint8_t *buf, size_t len) {
```

cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If PREFIX is a macro then please configure it.

(unknownMacro)